templates/deno/app/entry.client.tsx

```diff
@@ -1,3 +1,6 @@
+/// <reference no-default-lib="true" />
+/// <reference lib="dom" />
```

Comment:
I think this should be in a `tsconfig.json` instead? 🤔

CC/ @pcattori

Reply:
Other templates have it in its `tsconfig.json`.

I know @pcattori removed the file on purpose, but I can't remember the reasoning behind it.